indexbuilder.go

```diff
@@ -149,9 +149,17 @@ func (s *postingsBuilder) newSearchableString(data []byte, byteSections []Docume
 type IndexBuilder struct {
 	contentStrings  []*searchableString
 	nameStrings     []*searchableString
+	symbolStrings   []*searchableString
```

Comment:
why is this searchableString? And why are you adding this? I don't think we read this data since we rely on contentStrings.

Reply:
The `symbolStrings` is just a concatenated form of the ctags metadata for each symbol.

Format: `<uint32><uint32><uint32><uint32>`
                   Sym        Kind        Parent     ParentKind

It's not a "searchable" string, but I stored it as a searchableString so I could use the `writeStrings` function to store it in the index file.
```
func (s *compoundSection) writeStrings(w *writer, strs []*searchableString) {
	s.start(w)
	for _, f := range strs {
		s.addItem(w, f.data)
	}
	s.end(w)
}
```

Reply:
Got it. Rather just duplicate that function to make the type clearer.